world/models.py

```diff
@@ -68,13 +70,49 @@ class Location(SingletonModel):
         default="http://www.fema.gov",
         help_text="A link to your local office of emergency management."
     )
-    
+
     def __unicode__(self):
         return u"Location Information"
 
+    @staticmethod
+    def get_data_bounds():
+        bounds = {
+
+    ######################################################
+    # GENERATED CODE GOES HERE
+    # DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
+    # Location
+    # END OF GENERATED CODE BLOCK
+    ######################################################
+        }
+
+        # The smallest/largest possible values, as appropriate, so the map will display
+        # something if there is no data
+        north = [-180]
+        west = [-90]
+        south = [180]
+        east = [90]
+
+        for box in bounds.values():
+            west.append(box[0])
+            south.append(box[1])
+            east.append(box[2])
+            north.append(box[3])
+
+        # The largest box that contains all the bounding boxes, how Leaflet wants it.
+        return [[min(south), max(west)], [max(north), min(east)]]
+
```

Comment:
I think it should be min(west) and max(east) to get the biggest bounding box (remember that all continental US longitudes are negative because we're W of Greenwich).